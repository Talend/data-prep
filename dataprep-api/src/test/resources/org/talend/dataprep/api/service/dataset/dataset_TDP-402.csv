id;firstname;lastname;age;date-of-birth;alive
1;John;Lennon;40;27/09/1940;false
2;David;Bowie;67;14/08/1947;true
3;Dylan;Bob;74;empty;
4;Hendrix;Jimi;;1942-11-27;false