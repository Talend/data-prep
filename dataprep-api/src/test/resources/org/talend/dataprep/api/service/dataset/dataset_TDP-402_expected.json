{
  "records": [
    {
      "0000": "1",
      "0001": "John",
      "0002": "Lennon",
      "0003": "40",
      "0004": "27-09-1940",
      "0005": "false",
      "tdpId": 1
    },
    {
      "0000": "2",
      "0001": "David",
      "0002": "Bowie",
      "0003": "67",
      "0004": "14-08-1947",
      "0005": "true",
      "tdpId": 2
    },
    {
      "0000": "3",
      "0001": "Dylan",
      "0002": "Bob",
      "0003": "74",
      "0004": "empty",
      "0005": "",
      "0006": "",
      "tdpId": 3
    },
    {
      "0000": "4",
      "0001": "Hendrix",
      "0002": "Jimi",
      "0003": "",
      "0004": "27-11-1942",
      "0005": "false",
      "tdpId": 4
    }
  ]
}
